publ/image.py

```diff
@@ -304,7 +310,7 @@ def _image(self):
         return image
 
     # pylint:disable=too-many-arguments
-    def _render(self, path, size, box, flatten, kwargs, out_args):
+    def _render(self, path, crop, size, box, flatten, kwargs, out_args):
```

Comment:
The `LocalImage.render` argument list is getting comically long and in a future refactor I just want to separate the output filename generation out and only pass `kwargs` in, but that's outside the scope of this specific change so I'm mostly just rambling. :)